### PR TITLE
New package: teams-bin-1.3.00.5153

### PR DESCRIPTION
**Pull request by Piraty**
- [ ] restricted?
- [ ] license ??

### CODE REVIEW

File: srcpkgs/teams/template [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	vbin ${FILESDIR}/teams-wrapper.sh teams
+
+	vmkdir usr/lib/teams
+	vmkdir usr/share
+
+	cp -r usr/share/teams ${DESTDIR}/usr/lib/
+
+	for item in applications pixmaps ; do
+		cp -r usr/share/$item ${DESTDIR}/usr/share/$item
+	done
```
Comment (Anachron) [inline srcpkgs/teams/template:32]: If its just two items wouldn't it be better to remove the loop and replace it with single statements?

Comment (Johnnynator) [reply srcpkgs/teams/template:32]: Also why not use `vcopy` instead of `cp -r`.

**Comment by abenson** [comment]
This was my take on it. [teams-bin](https://github.com/abenson/void-packages/commit/f8ac51e309cd375cec2633f1564b8828c97dccca)

File: srcpkgs/teams/template [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Template file for 'teams'
+pkgname=teams
+version=1.3.00.5153
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz"
+short_desc="Official Microsoft Teams client"
+maintainer="Piraty <piraty1@inbox.ru>"
+license="custon:Proprietary"
```
Comment (fosslinux) [inline srcpkgs/teams/template:9]: `custom` not `custon`

**Comment by Piraty** [comment]
thanks for feedback, I resolved all issues

regarding license and distribution: [exherbo](https://git.exherbo.org/summer/packages/net-im/teams/index.html) labels it MIT licensed (the text found in LICENSE backs this), others ([nix](https://github.com/NixOS/nixpkgs/blob/release-20.03/pkgs/applications/networking/instant-messengers/teams/default.nix#L81), [repology](https://repology.org/project/teams/versions)) call it nonfree.
any idea?

**Comment by abenson** [comment]
If we're distributing the binary, still think it should be `-bin`. It's definitely not MIT, and I'm pretty sure its restricted.

In *ReadmeFirstTermsofUse.txt*,

> MICROSOFT SOFTWARE LICENSE TERMS
> MICROSOFT TEAMS FOR LINUX (PRE-RELEASE VERSION)
> *...*
> 6. SCOPE OF LICENSE. The software is licensed, not sold. Microsoft reserves all other rights. Unless applicable law gives you more rights despite this limitation, you will not (and have no right to):
> a) work around any technical limitations in the software that only allow you to use it in certain ways;
> b) reverse engineer, decompile or disassemble the software;
> c) remove, minimize, block, or modify any notices of Microsoft or its suppliers in the software;
> d) use the software in any way that is against the law or to create or propagate malware; or
> e) share, publish, distribute, or lend the software, provide the software as a stand-alone hosted solution for others to use, or transfer the software or this agreement to any third party.

File: srcpkgs/teams/files/teams-wrapper.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+LD_LIBRARY_PATH=/usr/lib/teams exec /usr/lib/teams/teams "$@"
```
Comment (ahesford) [inline srcpkgs/teams/files/teams-wrapper.sh:2]: Can `patchelf` not be used to set the `rpath` of the teams executable, rather than wrapping?

**Pull request closed without merging**